scripts/validate-nimbus-file.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+TARGET_PATH=/tmp/nimbus-cli
+NIMBUS_INSTALL_REMOTE_PATH=https://raw.githubusercontent.com/mozilla/application-services/main/install-nimbus-cli.sh
+NIMBUS_INSTALL_LOCAL_PATH=$TARGET_PATH/install.sh
+
+if [ -d $TARGET_PATH ]; then
```

Comment:
shellcheck might get you do `"${TARGET_PATH}"` because whitespace things

Reply:
Thanks — updated in [43b653b](https://github.com/mozilla/blurts-server/pull/4861/commits/43b653b0846d7313ceba8c61c510605682a413f6)!